src/pages/patientView/clinicalInformation/lib/clinicalAttributesUtil.js

```diff
@@ -132,6 +132,18 @@ function derive(clinicalData) {
                     caseTypeNormalized = 'cfDNA';
                 } else if (caseTypeLower.indexOf('prim') >= 0) {
                     caseTypeNormalized = 'Primary';
+                } else if (caseTypeLower.indexOf('ctdna') >= 0) {
+                    caseTypeNormalized = 'ctDNA';
```

Comment:
@BasLee i feel like ctDNA/cfDNA are used interchangeably - altho ctDNA should only refer to tumor DNA you usually only know what the ctDNA part of the cfDNA is during analysis afaik. Not really when the sample is collected. Should we harmonize them to be the same `caseTypeNormalized`, e.g. always use `cfDNA`? Or use the same color?